test/unit/messagesTest.ts

```diff
@@ -171,7 +171,7 @@ describe('Messages', () => {
       expect(importMessageFileStub.secondCall.args[1]).to.equal(path.join(expectedMsgDirPath, msgFiles[1]));
     });
 
-    it('should not remove the "/lib" if dist is already removed', () => {
+    it.skip('should not remove the "/lib" if dist is already removed', () => {
```

Comment:
Why the skip?

Reply:
Oops... removed it.

I do it locally cuz this doesn't work on Windows for some reason... causes a JavaScript heap overload. Pretty sure it's something to do with the paths cuz there are another like 10 tests that fail because of path reasons on Windows.
![image](https://user-images.githubusercontent.com/1084688/127369886-a3c704a0-ef05-4cd7-bdd6-130b0a5af503.png)